docs/source/tutorials/linalg/linear_solvers_quickstart.ipynb

```diff
@@ -0,0 +1,2673 @@
+{
```

Comment:
<p>Capitalization in header: Solver</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>done</p>